src/main/scala/org/opennetworkinsight/ProxyPostLDA.scala

```diff
@@ -74,48 +51,73 @@ object ProxyPostLDA {
         "proxy_csbytes",
         "proxy_fulluri")
 
-      df_cols = df.columns
-      val tempRDD: org.apache.spark.rdd.RDD[String] = df.map(_.mkString(","))
-      tempRDD
-    }
+    logger.info("Computing conditional probability")
+    val scoredDF : DataFrame = score(sc, dataFrame, topicCount, ipToTopicMix, wordsToProbPerTopic)
+
+    val filteredDf = scoredDF.filter("score < " + threshold)
```

Comment:
Would be great to keep consistency across names, even establish our names convention for RDDs and DF. In this case I think filteredDf would look better as filteredDF. 

Not a big deal.